src/Tizen.TV.UIControls.Forms/GridView.cs

```diff
@@ -0,0 +1,145 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections;
+using Xamarin.Forms;
+
+namespace Tizen.TV.UIControls.Forms
+{
+    public class GridView : View
+    {
+        public static readonly BindableProperty ItemsSourceProperty = BindableProperty.Create(nameof(ItemsSource), typeof(IEnumerable), typeof(GridView), null);
+
+        public static readonly BindableProperty ItemTemplateProperty = BindableProperty.Create(nameof(ItemTemplate), typeof(DataTemplate), typeof(ItemsView<GridView>), null, validateValue: (b, v) => ((GridView)b).ValidateItemTemplate((DataTemplate)v));
+
+        public static readonly BindableProperty ItemHeightProperty = BindableProperty.Create("ItemHeight", typeof(int), typeof(GridView), -1);
+
+        public static readonly BindableProperty ItemWidthProperty = BindableProperty.Create("ItemWidth", typeof(int), typeof(GridView), -1);
+
+        public static readonly BindableProperty ItemHorizontalAlignmentProperty = BindableProperty.Create("ItemHorizontalAlignment", typeof(double), typeof(GridView), 0.0);
+
+        public static readonly BindableProperty ItemVeticalAlignmentProperty = BindableProperty.Create("ItemVeticalAlignment", typeof(double), typeof(GridView), 0.0);
+
+        public static readonly BindableProperty ThemeStyleProperty = BindableProperty.Create("ThemeStyle", typeof(string), typeof(GridView), default(string));
```

Comment:
I can not support `On<T>()` method properly because PlatformConfigurationRegistry class is internal in X.F. 
`
        public IPlatformElementConfiguration<T, GridView> On<T>() where T : IConfigPlatform
        {
            return null;
        }
`
Do you have any idea to solve this problem? 
Thank you in advance.